pkg/fileservice/aliyun_sdk.go

```diff
@@ -53,18 +58,6 @@ func NewAliyunSDK(
 		return nil, err
 	}
 
-	// env vars
```

Comment:
Does this work for RRSA after removing this?

Reply:
OIDC credentials will be loaded in the default provider at this line: https://github.com/matrixorigin/matrixone/pull/14116/files#diff-53c458acf1a3339cd979fff55fe0f53e01073eadae1e21b997ff1933461b6539L632